pandas/tests/core/dtypes/test_dtypes.py

```diff
@@ -356,6 +357,70 @@ def test_not_string(self):
         self.assertFalse(is_string_dtype(PeriodDtype('D')))
 
 
+class TestCategoricalDtypeParametrized(object):
```

Comment:
I think we can just call this `TestCategoricalDtype` (consistent with other class names)